lib/bunch/sref.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Bunch.Sref do
```

Comment:
The name doesn't mean much. When I saw a notification with `add sref` I thought it's some format specific field in something like opus. Or secure ref. The point is: it does not describe that it is well. 
Ideas for alternatives: `Hash(ed)Ref`, `ShortRef`